Index: Presentation/appsettings.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>{\r\n  \"Logging\": {\r\n    \"LogLevel\": {\r\n      \"Default\": \"Warning\",\r\n      \"Microsoft.AspNetCore\": \"Warning\"\r\n    }\r\n  },\r\n  \"DbOptions\": {\r\n    \"DefaultConnection\": \"Host=127.0.0.1;Port=5433;Database=postgres;Username=postgres;Password=root123\"\r\n  },\r\n  \"AllowedHosts\": \"*\"\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Presentation/appsettings.json b/Presentation/appsettings.json
--- a/Presentation/appsettings.json	(revision 1915436fa5f23e992b613747ef8c3aface39f855)
+++ b/Presentation/appsettings.json	(date 1760947446124)
@@ -6,7 +6,7 @@
     }
   },
   "DbOptions": {
-    "DefaultConnection": "Host=127.0.0.1;Port=5433;Database=postgres;Username=postgres;Password=root123"
+    "DefaultConnection": "Host=127.0.0.1;Port=5432;Database=postgres;Username=postgres;Password=root123"
   },
   "AllowedHosts": "*"
 }
